### PR TITLE
Project 3: Cloning "NewsWeek" website using BootStrap.

### PR DESCRIPTION
**Pull request by FdI96**
This is a clone of the "NewsWeek" webpage.
The goal is to create a similar website using mainly Boostrap 5.0v and make the site responsive in three different screens (Mobile version: <= 767px ; Tablet or medium screen size: >=768px and <= 991 ; Desktop or big screen size: >= 992).

We used https://getbootstrap.com/ documentation to build this website.

This project is part of the first module in the third week of the Microverse Program.

This project was made by two collaborators:
@widzthedvloper and @FdI96.

### CODE REVIEW

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED ♻️ 🛑 
Hello, @FdI96 and @widzthedvloper this is good progress. Your project requires a bit of work but you have done a good job so far. Kindly fix the following:

### DESKTOP VIEW:

- [x] At the top bar your items are not well aligned. Kindly compare them with the actual news website and you can tell there are some alignment issues. Kindly check out this [link](https://getbootstrap.com/docs/4.0/utilities/flex/) to learn more on bootstrap alignment.
  **yours**
menu yours
 **original**
menu original

- [x] Your main navigation bar does not resemble the original. Kindly fix this
   **yours**
main nav yours
**original**
main nav original

- [x]  You don't have a featured slideshows section. Kindly add this section to your webpage.
     
featured

### MOBILE VIEW

- [x] Your mobile view is perfect. Good work on this. However, just as stated above remember to add the featured-slide section

### if you have any question kindly reach me on slack @peterrobert

### HAPPY CODING 💻

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle:

### Hi @FdI96 and @widzthedvloper  ! 👋 👋 

### You have done well so far :clap: However, there are changes required for you to make in order for this project to be approved. Kindly work on them and submit them for another code review.


**General**
- [x] Your project looks great. However, your project cannot be approved until you write a descriptive PR Message. It's essential that your repository is professional, as this is one of the things that potential colleagues will look at when recruiting team members. Please refer to [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more information.


**Recommended**

- [x] On desktop screens, your page is taking the full width of the screen, refer to the actual site for more clarity.
- [x] Fix your navigation bar to look as similar as possible to the actual site's.
- [x] Your vertical scroll is missing on the page, making it compulsory to navigate the page by the arrow keys, please inspect this issue.
- [x] The more stories sections is stuck under the next section please review this.

**Your clone**
Screenshot (159)

- [x] Fix the following sections and refer to the [Newsweek](newsweek.com) site for more clarity: 
- In the Magazine
- Editor's Picks
- The section before the footer
- The footer


__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1476 @@
...
+                            <div class="weather container"><a href="#"><img
+                                        src="https://g.newsweek.com/img/weather/28.png" alt="cloud"><span
+                                        class="text-white">28° Corrientes, AR
+                                        &gt;</span></a>
+                            </div>
+                            <div class="date container"><span>Mon, Dec 14, 2020</span></div>
+                        </div>
+                        <div
+                            class="news-week pt-2 col-lg col-6 d-flex justify-content-start justify-content-lg-center align-items-lg-center w-50 w-lg-100">
+                            <img class="d-none d-lg-inline mb-2" src="images/nav-big-logo.png" alt="">
```
Comment (umairarshadbutt) [inline index.html]: Please add the description for the alt attribute

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1476 @@
...
+                                    </path>
+                                </g>
+                            </svg>
+                        </div>
+                        <div class="icons col-lg col-6 d-flex flex-row justify-content-end mb-lg-3">
+                            <h4 class="align-self-lg-end pe-4 pt-1 d-none d-md-inline my-0 text-white fw-bold">LOGIN
+                            </h4>
+                            <i class="login-logo fa fa-user pt-2 pe-2 h-100 d-md-none"></i>
+                            <button class="subscribe ps-3 pe-3 fw-bold align-self-lg-end">SUBSCRIBE&gt;</button>
+                            <img class="h-100 d-lg-none" src="images/navbar/hamburger.png" alt="">
```
Comment (umairarshadbutt) [inline index.html]: Please add the description for the alt attribute

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1476 @@
...
+                <div class="big-row container-fluid">
+                    <div class="row-sm row d-flex d-md-none">
+                        <div class="top-story col-12 col-md-6">
+
+                            <div class="title">
+                                <h6>TOP STORY</h6>
+                            </div>
+                            <div class="top-story-card-img position-relative">
+                                <a href="#">
+                                    <img style="width: 100%;" src="images/topstory/topstoryimg.webp" alt="joebiden">
```
Comment (umairarshadbutt) [inline index.html]: Please remove the inline style

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1476 @@
...
+                        <div class="combination1 overflow-scroll col-md-6 col-lg-4">
+
+                            <div class="top-story">
+
+                                <div class="title">
+                                    <h6>TOP STORY</h6>
+                                </div>
+                                <div class="top-story-card-img position-relative">
+                                    <a href="#">
+                                        <img style="width: 100%;" src="images/topstory/topstoryimg.webp" alt="joebiden">
```
Comment (umairarshadbutt) [inline index.html]: Please remove the inline style

File: index.html [hunk 5]
```diff
@@ -0,0 +1,1476 @@
...
+                            </path>
+                        </g>
+                    </svg>
+                    <p class="align-self-md-end mb-md-0" style="font-size: 12px;">© 2020 NEWSWEEK DIGITAL LLC</p>
```
Comment (umairarshadbutt) [inline index.html]: Please remove the inline style

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @FdI96 and @widzthedvloper 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Please implement the suggestions as previous TSEs suggested
- [x] Remove inline styles from HTML file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
- [x] Please add the description in the `alt` attribute. (See inline comment)

### Navbar
- [x] Please fix the issues for the navbar as previous TSEs has mentioned. You can visit [Newsweek](https://www.newsweek.com/)

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://rawcdn.githack.com/FdI96/Bootstrap-Project/08ef5540331d1b56ecf4699941a1f72ce39a98a2/index.html)|
|:---:|:---:|
|source-page|clone|

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know by leaving a comment here and mentioning me. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Comment by descholar-ceo** [comment]
# This Pull Request is under review by [descholar-ceo](https://github.com/descholar-ceo) :eyes: :eyes: 
## Please wait a bit ...

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @FdI96 and @widzthedvloper  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's beacuse this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [x] Fix the logo in navbar on mobile screen, it is very small compared to the original
- [x] And reduce the space which is on the right most of the navbar, between the hamburger bars and the screen edge (on the right side) 

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-12-18 02-21-30|Screenshot from 2020-12-18 02-22-15|

This screenshot is taken on screens <550px

:point_right: Could you see the difference :question: :question: :question: 

- [x] Images in the opinion section are strecthed, fix that issue

| Your clone | Original page |
| --- | --- |
|Screenshot from 2020-12-18 02-27-42 |Screenshot from 2020-12-18 02-28-12 |

:point_right: Could you see the difference :question: :question: :question: 

- [x] Fix the Signup form, on mobile screens

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-18 02-29-34 | Screenshot from 2020-12-18 02-30-39 |

:point_right: Could you see the difference :question: :question: :question: 

- [x] Fix the news letter section on mobile screens

| Your clone | Original page |
| --- | --- |
| Screenshot from 2020-12-18 02-32-55 | Screenshot from 2020-12-18 02-33-59|

:point_right: Could you see the difference :question: :question: :question: 
___
_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_

**Comment by mimipeshy** [comment]
## STATUS: APPROVED

Hey team 👋 👋 , great job on working on all required changes, this project is therefore approved. Go ahead and merge 😄 
All the best in the next projects. Happy Coding 💻 

[OPTIONAL]

- [ ] Images in the opinion section are stretched, fix that issue 

Regards
Peris Ndanu

**Pull request merged**